src/amo/components/HeroRecommendation/index.js

```diff
@@ -184,6 +195,51 @@ export class HeroRecommendationBase extends React.Component<InternalProps> {
       loading = true;
     }
 
+    const renderHeroTitle = () => {
+      // translators: If uppercase does not work in your locale,
+      // change it to lowercase. This is used as a secondary heading.
+      let titleText = i18n.gettext('SPONSORED');
+
+      const promotedCategory = _getPromotedCategory({
+        addon,
+        clientApp,
+        forBadging: true,
+      });
+
+      if (promotedCategory === RECOMMENDED) {
+        // translators: If uppercase does not work in your locale,
+        // change it to lowercase. This is used as a secondary heading.
+        titleText = i18n.gettext('RECOMMENDED');
+      } else if (promotedCategory === LINE) {
+        // translators: If uppercase does not work in your locale,
+        // change it to lowercase. This is used as a secondary heading.
+        titleText = i18n.gettext('BY FIREFOX');
+      }
+
+      return (
+        <div className="HeroRecommendation-title">
+          <div className="HeroRecommendation-title-text">
+            {loading ? <LoadingText width={20} /> : titleText}
+          </div>
+          {![LINE, RECOMMENDED].includes(promotedCategory) ? (
```

Comment:
do we want to show the link when `loading` is true?  (no?)

Reply:
Aha, good catch. No, we do not.